modules/setup/application/forms/LdapDiscoveryPage.php

```diff
@@ -74,6 +74,8 @@ public function isValid($data)
                 $this->discovery = Discovery::discoverDomain($data['domain']);
                 if ($this->discovery->isSuccess()) {
                     return true;
+                } else {
+                    $this->addError($this->discovery->getError()->getMessage());
```

Comment:
Please use Form::error instead since it's ours and generally used in the Wizard. Form::addError is from Zend and may differ in style.